llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -1180,6 +1180,9 @@ enum : unsigned {
 
 // Section flags.
 enum : unsigned {
+  // No flag
+  SHF_NONE = 0x0,
```

Comment:
Does the standard define this value? Maybe should avoid introducing names that aren't used in it

Reply:
Strong +1 to this. `SHF_NONE` isn't a standard value, so we shouldn't list it here.

Furthermore, these are bitwise flags, having a definition for a flag with value 0 makes no sense, since something like `SHF_NONE | SHF_WRITE` would be meaningless.

Reply:
OK, see a similar discussion https://reviews.llvm.org/D61046 before too.